app/(nms)/network-configuration/page.tsx

```diff
@@ -28,93 +29,152 @@ const NetworkConfiguration = () => {
   const { data: networkSlices = [], isLoading: loading, status: networkSlicesQueryStatus, error: networkSlicesQueryError } = useQuery({
     queryKey: [queryKeys.networkSlices, auth.user?.authToken],
     queryFn: () => getNetworkSlices(auth.user ? auth.user.authToken : ""),
-    enabled: auth.user ? true : false,
-    retry: (failureCount, error): boolean => {
-      if (error.message.includes("401")) {
-        return false
-      }
-      return true
-    }
+    enabled: !!auth.user, // Only enable if the user is authenticated
```

Comment:
```suggestion
    enabled: Boolean(auth.user),
```

This would be clearer.

Reply:
Done.